menoh/dtype.hpp

```diff
@@ -38,10 +45,30 @@ namespace menoh_impl {
     struct dtype_to_type {};
 
     template <>
-    struct dtype_to_type<dtype_t::float_> {
+    struct dtype_to_type<dtype_t::float32> { // including dtype_t::float_
         using type = float;
     };
 
+    template <>
+    struct dtype_to_type<dtype_t::int8> {
+        using type = std::int8_t;
+    };
+
+    template <>
+    struct dtype_to_type<dtype_t::int16> {
+        using type = std::int16_t;
+    };
+
+    template <>
+    struct dtype_to_type<dtype_t::int32> {
+        using type = std::int32_t;
+    };
+
+    template <>
+    struct dtype_to_type<dtype_t::int64> {
+        using type = std::int64_t;
+    };
+
```

Comment:
Don't you add support for `dtype_t::float16` and `dtype::float64` here?

Reply:
Here, we can add `float16` and `float64` so I will do. Thanks for pointing out.